TestModels/dafny-dependencies/StandardLibrary/runtimes/go/ImplementationFromDafny-go/UTF8/externs.go

```diff
@@ -41,16 +34,27 @@ func Encode(utf16EncodedDafnySeq dafny.Sequence) Wrappers.Result {
 	decodedUtf16 := utf16.Decode(encodedUtf16)
```

Comment:
Talked offline. We don't require valid check here because this is only called from dafny side where we trust that the character is valid.